Snippets/Testing/Testing_6/Logging/LoggingTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using NServiceBus.Logging;
+using NServiceBus.Testing;
+using NUnit.Framework;
+
+[TestFixture]
+public class LoggingTests
+{
+    #region LoggerTesting
+    [Test]
+    public async Task ShouldLogCorrectly()
+    {
+        var logStatements = new StringBuilder();
```

Comment:
would it be better to keep a list of statements?

Reply:
Not following, do you mean something likt this https://stackoverflow.com/questions/10303315/convert-stringwriter-to-string ?

Reply:
as in collect the log statements in a `list<string>`

Reply:
Couldn't find a way to get a string writer to populate a list of string? Or are U saying to change the `TestingLoggerFactory` to capture to a list instead of writing to a `TextWriter` ?

Reply:
i think u would need to change `TestingLoggerFactory` to allow that.

Reply:
Raised an issue to track https://github.com/Particular/NServiceBus.Testing/issues/129 that potential change